docs/ReleaseNotes.md

```diff
@@ -1,6 +1,66 @@
 # Acuminator Release Notes
 This document provides information about fixes, enhancements, and key features that are available in Acuminator.
 
+## Acuminator 2.1
+Acuminator 2.1 includes the diagnostics, code fixes, enhancements, and bug fixes described in this section, as well as the features that have been implemented in previous versions.
+
+### New Diagnostic and Code Fixes
+In this version, the diagnostic and code fixes have been added for the following issue.
+
+| Code                            | Issue Description                                                | Type    | Diagnostics | Code Fix    | 
+| ------------------------------- | ---------------------------------------------------------------- | ------- | ----------- | ----------- |
+| [PX1007](diagnostics/PX1007.md) | A public class should have a description in the `summary` XML tag. | Warning (Level 3: Informational)	 | Available | Available   |
+
+### Ability to Suppress a Diagnostic by Adding a Code Comment
+In Acuminator 2.1, you can suppress any diagnostic by adding a specific comment directly in code. To add a suppression comment, you select **Suppress the PX10YY diagnostic with Acuminator > in a comment** (where PX00YY is the code of the diagnostic) as a code fix for the diagnostic, as shown in the following screenshot.
+
+![Suppression in Comment](images/SuppressDiagnosticInComment.png)
+
+The code below shows an example of a suppression comment that has been added by the code fix. A suppression comment includes the following parts:
+
+ - The code of the diagnostic (`PX1007` in the code example below).
+ - The name of the diagnostic (`NoXmlCommentForPublicClass` in the following code example), which can give a code reviewer an idea of what the suppressed diagnostic is about.
+ - The justification of the suppression, which explains why this diagnostic is suppressed for this code fragment. You type the explanation instead of `[Justification]`.
+
+```C#
+// Acuminator disable once PX1007 NoXmlCommentForPublicClass [Justification]
+public class ARReleaseProcess_Extension : PXGraphExtension<ARReleaseProcess>
+{
+    ...
+}
+```
+
+You can also suppress an Acuminator diagnostic in the Acuminator suppression file. This option is now available as a code fix and has been removed from the menu commands. For details about the diagnostic suppression, see [Diagnostic Suppression](diagnostics/DiagnosticSuppression.md).
+
+### Enhancements and Bug Fixes in the Code Map
+Acuminator 2.1 includes the following enhancements in the code map:
+
+ - The code map now can be used for DACs and DAC extensions. For each DAC, the list of key fields and the full list of DAC fields is displayed. For each DAC field, the code map shows the list of attributes. The code map for a file that contains two DACs is shown in the following screenshot.
+
+ ![Code Map](images/CodeMapForDacs.png)
+
+ - The code map now supports cycling navigation between partial definitions.
```

Comment:
@k-s-popova only for a graph. The DAC shouldn't have two partial declarations as Acumatica depends on the order of fields and it it unspecified

Reply:
Fixed